raimitigations/dataprocessing/sampler/rebalance.py

```diff
@@ -227,8 +231,8 @@ def test_if_float(value):
         if has_null:
             raise ValueError(
                 f"ERROR: the column {self.label_col_name} provided to the 'rebalance_col' parameter contains "
-                + f"null values. The 'rebalance_col' only accepts columns with integer values, which represents the "
-                + f"classes to be rebalanced."
+                + f"null values. The 'rebalance_col' only accepts columns with non-float values, which represents "
```

Comment:
What non-float types exactly?

Reply:
Non-float type is anything that is not a float. Basically, it means integer and string type columns, because these two types can be easily converted into classes. Maybe replace the "non-float values" with "integer or string values"?

Reply:
Yeah, I think being explicit would be clearer. (You exclude Boolean, for example.)